sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md

```diff
@@ -316,6 +316,35 @@ spring:
           useInsecureTrustManager: true
 ```
 
+You can also enable the application to obtain the latest version of the certificate on the portal
+when the inbound certificate is not recognized by using the configuration below.
```

Comment:
When the inbound certificate is not trusted, the `KeyVaultKeyStore` can fetch certificates from KeyVault if the following property is configured: